### PR TITLE
Change pipeline sample to use Node.js+MongoDB sample

### PR DESCRIPTION
**Pull request by sspeiche**
Since we don't have a ruby slave, we should leverage the nodejs slave we have

### CODE REVIEW

**Comment by bparees** [comment]
there's really no point.  even if we use the nodejs slave, we're not actually executing any build operations on the slave or leveraging any platform specifics from the slave.  The build operations happen in openshift (we just trigger a ruby/nodejs/whatever buildconfig in openshift).

so with that in mind do you still want the example updated?

**Comment by sspeiche** [comment]
Understand on the slave point, was just trying to align the pipeline sample with slaves we have. A common source of confusion I've seen where people see the Ruby sample, the preference is node or java. This PR was to help with that confusion and give a better starting example.

File: examples/jenkins/pipeline/samplepipeline.json [hunk 1]
```diff
@@ -43,463 +43,456 @@
...
-      },
-      "spec": {
-        "ports": [
-          {
-            "name": "web",
-            "protocol": "TCP",
-            "port": 5432,
-            "targetPort": 8080,
-            "nodePort": 0
+
```
Comment (bparees) [inline examples/jenkins/pipeline/samplepipeline.json]: delete the extra space

File: examples/jenkins/pipeline/samplepipeline.json [hunk 2]
```diff
@@ -43,463 +43,455 @@
         "strategy": {
           "type": "JenkinsPipeline",
           "jenkinsPipelineStrategy": {
-            "jenkinsfile": "node('maven') {\nstage 'build'\nopenshiftBuild(buildConfig: 'ruby-sample-build', showBuildLogs: 'true')\nstage 'deploy'\nopenshiftDeploy(deploymentConfig: 'frontend')\n}"
+            "jenkinsfile": "node('nodejs') {\nstage 'build'\nopenshiftBuild(buildConfig: '${NAME}' showBuildLogs: 'true')\nstage 'deploy'\nopenshiftDeploy(deploymentConfig: '${NAME}')\n}"
```
Comment (dongboyan77) [inline examples/jenkins/pipeline/samplepipeline.json]: should add a comma before 'showBuildLogs'

Comment (bparees) [inline examples/jenkins/pipeline/samplepipeline.json]: thanks @dongboyan77 

@sspeiche you said you tested this....

Comment (sspeiche) [inline examples/jenkins/pipeline/samplepipeline.json]: @bparees I guess not enough, I did deploy the template but I guess I was impatient to wait for Jenkins to run (walks away hanging head in shame)

Comment (bparees) [inline examples/jenkins/pipeline/samplepipeline.json]: fixing here https://github.com/openshift/origin/pull/11576

**Pull request merged**